client/src/components/Custom/LabeledInput.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+function LabeledInput({ className, name, label, type, placeholder }) {
+    return (
+        <div className={className}>
+            <div className="info-group mb-3">
+                <label>{label}</label>
+                <input
+                    name={name}
+                    type={type}
+                    className="info-input"
+                    placeholder={placeholder}
```

Comment:
will have to do a check for if the placeholder is undefined in the case that a placeholder value is not passed into the component

Reply:
Hey Elsie, what do you mean by this? like if input is empty, give an error message or something?

Reply:
yeah so I was wondering if a placeholder isn't passed in, will the labeled input component break? if it does, then there should probably be a check for that, but considering how close to deadline it is right now, maybe just write a todo note for yourself and then i can resolve this comment

Reply:
yea, I was looking into writing an error message for it, I'll see how much I can get done.